src/features/SwapAssets/components/ConfirmSwap.tsx

```diff
@@ -57,8 +57,15 @@ export default function ConfirmSwap({
         toAmount={toAmount.toString(10)}
       />
       <FromToAccount
-        from={{ address: account.address, label: account.label }}
-        to={{ address: account.address, label: account.label }}
+        fromAccount={{
+          address: account.address,
+          addressBookEntry: { ...account, notes: '', network: account.networkId }
```

Comment:
Won't this possibly remove notes for existing accounts if updating the label?

Reply:
Same for below

Reply:
i made a change that should cover this as long as im not dumb xD. please take a look!